certsuite/harness.py

```diff
@@ -421,6 +420,49 @@ def check_device_profile(device_profile_path):
                     logger.error('Invalide device profile file [%s]' % device_profile_path)
```

Comment:
just catch that you can merge
'''python
else:
    if not 'contact' in device_profile_object['result']:
'''
to
'''python
elif not 'contact' in device_profile_object['result']
'''